pandas/core/resample.py

```diff
@@ -444,7 +453,14 @@ def _groupby_and_aggregate(self, how, *args, **kwargs):
             #  a DataFrame column, but aggregate_item_by_item operates column-wise
             #  on Series, raising AttributeError or KeyError
             #  (depending on whether the column lookup uses getattr/__getitem__)
-            result = grouped.apply(how, *args, **kwargs)
+            with rewrite_warning(
```

Comment:
There are a few similar code blocks like this. Is it possible to group to either put them in a function or maybe catch/rewrite the error in `pandas/core/groupby/groupby.py` around line 3668?

Reply:
Done.